app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 in_class: int,
+                 in_mark: int,
+                 in_brand: str) -> None:
+        self.comfort_class = in_class
+        self.clean_mark = in_mark
+        self.brand = in_brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 in_distance: float,
+                 in_power: int,
+                 in_rating: float,
+                 in_count: int) -> None:
+        self.distance_from_city_center = in_distance
+        self.clean_power = in_power
+        self.average_rating = in_rating
+        self.count_of_ratings = in_count
+
+    def calculate_washing_price(self, in_car: Car) -> float:
+        price = in_car.comfort_class                    \
+            * (self.clean_power - in_car.clean_mark)    \
+            * self.average_rating                       \
+            / self.distance_from_city_center
+        price = round(price, 1)
+        return price
+
+    def wash_single_car(self, in_car: Car) -> None:
+        in_car.clean_mark = self.clean_power
```

Comment:
According to README.md for this method next condition exits: ```if wash_station.clean_power is greater than car.clean_mark```

Reply:
NO. "car should have clean_mark equals wash station's clean_power, if wash_station.clean_power is greater than car.clean_mark". but whether condition meet  or not - could be realised it other method.

Reply:
Agree

Reply:
You need to check if clean_power is greater than clean_mark here because if you run this method separately from others it will not work correctly